lib/data/repositories/activity_repository_impl.dart

```diff
@@ -41,12 +41,8 @@ class ActivityRepositoryImpl
   @override
   Future<Either<Failure, List<Activity>>> getActivities() async {
     try {
-      final either = await _source.getActivities();
-      return either.fold((failure) {
-        return Left(failure);
-      }, (activities) {
-        return Right(_activityAdapter.modelsToEntities(activities));
-      });
+      final activities = await _source.getActivities();
+      return Right(_activityAdapter.modelsToEntities(activities));
```

Comment:
rebase or merge or something on master to change the results using the `Success` format